### PR TITLE
[RETINA-197] Password Reset

### LINKED ISSUE #197: Password Reset Page

### PR DESCRIPTION
**Pull request by Jbcampbe**
Closes #197 

image

### CODE REVIEW

**Comment by ekraihan** [comment]
Beautiful notifications! ;)

**Comment by ekraihan** [comment]
@Jbcampbe 
Notes:
- Since it takes so long for the email to be sent, I wondered about some kind of "loading" state?
- I wondered if the email success popup could say "Instructions *have* been sent" ? ;)
- On the email request dialogue popup, could you have the text "email address" be more grayed out, like we added on the add tool page?
- I wondered if the error message on failed-password-reset-request could include "make sure you typed in the correct email"

File: src/main.js [hunk 1]
```diff
@@ -113,7 +126,8 @@ const dictionary = {
       serialNumber: 'serial number',
       modelYear: 'model year',
       purchasedFrom: 'purchased from',
-      price: 'price'
+      price: 'price',
```
Comment (ekraihan) [inline src/main.js]: Is this dictionary for in case we go international? :)

Comment (Jbcampbe) [reply src/main.js]: it means that if we go international then we already have a validation library that supports it : )

**Pull request merged**